config/crd/bases/infrastructure.cluster.x-k8s.io_oscclusters.yaml

```diff
@@ -214,9 +214,21 @@ spec:
                                 type: string
                             type: object
                           type: array
-                        subnetname:
+                        subnets:
                           description: The subnet tag name associate with a Subnet
-                          type: string
+                          items:
+                            properties:
+                              ipSubnetRange:
+                                description: Subnet Ip range with CIDR notation
+                                type: string
```

Comment:
Do you want all of this ? could we just have subnets ref ?

Reply:
@outscale-mdr 
I can have:
1) an array of string
2) an array of a new OscSubnet with only name.
3)  an arrayf of OscSubnet with optional (not use) parameters.

I choose the third solution to handle the same way as routeTable, publicIp, securityGroup.
What do you think ?

Reply:
Thank you fir the answer, you mean that potentially, the user could add values like `ipSubnetRange` and it will not be use at all ?

Reply:
Yes

Reply:
It is counter-intuitive, don't you think ?

Reply:
Ok i change it